tabulator/helpers.py

```diff
@@ -23,7 +20,9 @@ def detect_scheme(source):
     For example `http` from `http://example.com/table.csv`
 
     """
-    if hasattr(source, 'read'):
+    if isinstance(source, list):
```

Comment:
Here we need something smarter to work not only with lists but any iterables. 
May be we should use `NativeLoader` for all non string sources.